content/doc/book/system-administration/backing-up.adoc

```diff
@@ -1,211 +0,0 @@
----
```

Comment:
did you intend to delete this?

Reply:
Yes, it was from a different PR and I have no idea how I sucked it into this one